controllers/appsv1/deployment_webhook.go

```diff
@@ -132,7 +132,7 @@ func (d *deploymentInterceptor) Handle(ctx context.Context, req admission.Reques
 
 		const msg = "no suitable Jaeger instances found to inject a sidecar"
 		span.AddEvent(msg)
-		logger.Debug(msg)
+		logger.V(-1).Info(msg)
```

Comment:
what -1 means?

Reply:
With `logr` you don't have the semantic verbosity levels. You only have "numbers". When using `logr` with `zap`, they recommend to map the `zap` semantic with those numbers https://github.com/uber-go/zap/blob/e751939b68425b6aedd6bbf93704363c482880da/zapcore/level.go#L37-L52